### ISSUE TITLE
ReferenceError: filename is not defined

### ISSUE THREAD

**Issue by ivanoats** [swc-project/swc-loader#5]
I'm getting an error when trying to follow the example with webpack:
```
> ./node_modules/.bin/webpack
Hash: f728d32ddf1c4679aa5c
Version: webpack 4.29.0
Time: 102ms
Built at: 01/21/2019 5:42:22 PM
  Asset      Size  Chunks             Chunk Names
main.js  3.99 KiB    main  [emitted]  main
Entrypoint main = main.js
[./src/index.js] 226 bytes {main} [built] [failed] [1 error]

ERROR in ./src/index.js
Module build failed (from ./node_modules/swc-loader/src/index.js):
ReferenceError: filename is not defined
    at Object.<anonymous> (/Users/ivan/dev/swc-static/node_modules/swc-loader/src/index.js:26:13)
```
The full repo to reproduce the error is available here: https://github.com/ivanoats/swc-webpack-react-demo

I'm excited to start using this, looks like a great project.

**Comment by kdy1** [comment]
Fixed with version swc-loader@0.1.2 and swc@1.0.0-beta.22

See https://github.com/ivanoats/swc-webpack-react-demo/pull/1

**Comment by ivanoats** [comment]
Wow, thank you so much!
